_posts/2018-07-14-del_dotto_historic_tasting.markdown

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title:  "Del Dotto Historic Tasting and Tour"
+date:   2018-07-14 15:20:00 -0400
+categories: review tour tasting
+---
+Del Dotto ($75): http://www.deldottovineyards.com
+
+For me, Del Dotto is a solid introduction to the nicer Napa wine country experiences.  It has currently two locations (their historic site and St. Helena site), with a third coming online later this year.  My recommendation is to do the cellar barrel tasting at either of the two aforementioned sites: the historic site has a rustic feel to its chiseled-out caves, while the St. Helena site has a sophisticated, old, Italian feel.  Arrive 5-10 min before your appointment and you’ll enjoy an aperitif (likely their Sauvignon Blanc) before the tasting begins along with some nice photo opportunities.
```

Comment:
no comma in front of `while the st. helena`

no commas in `sophisticated, old, italian feel`